backend/backend.go

```diff
@@ -16,6 +16,7 @@ var (
 	ErrNotInitialized        = errors.New("backend not initialized")
 	ErrWorkItemLockLost      = errors.New("lock on work-item was lost")
 	ErrBackendAlreadyStarted = errors.New("backend is already started")
+	ErrSkipExistInstance 	 = errors.New("Skip creating existing instance")
```

Comment:
nit: golang style guides say that error messages should start with a lowercase letter. I see warnings in VS Code when I try to create error messages that start with capital letters.